boa/environment.py

```diff
@@ -470,9 +470,20 @@ class FakeMessage(Message):  # Message object with settable attrs
         return self.vm.state.computation_class.apply_message(self.vm.state, msg, tx_ctx)
 
     # function to time travel
-    def time_travel(self, sleep_time):
-        self.vm.patch.timestamp += sleep_time
-        self.vm.patch.block_number += sleep_time // 12
+    def time_travel(self, seconds=None, blocks=None, block_delta=12):
+
+        if seconds is None and blocks is None:
+            raise ValueError("One of seconds or blocks should be set")
+        if seconds is not None:
+            if seconds <= 0:
```

Comment:
i think time traveling to the past should be allowed

Reply:
Hmm can you give me an example where this would be ok?